cli/src/torii/utils.rs

```diff
@@ -49,10 +49,13 @@ pub mod body {
 
     /// Recover from failure in `versioned`
     pub fn recover_versioned(rejection: Rejection) -> Result<impl Reply, Rejection> {
-        if let Some(error) = rejection.find::<VersionError>() {
-            return Ok(error.into_response());
-        }
-        Err(rejection)
+        rejection
+            .find::<VersionError>()
+            .map(|e| {
+                // generally don't implement a trait for &Error
+                e.into_response()
+            })
```

Comment:
```suggestion
            .map(SomeTrait::into_response)
```

maybe?

Reply:
I'm not sure why this change shows up in my PR, maybe it was touched by LSP. But I agree. Maybe `ReponseExt::into_reponse`?